earth_enterprise/src/fusion/autoingest/SConscript

```diff
@@ -176,7 +176,7 @@ commoncpp = [
     'LayerConfigImpl.cpp', 'MapProjectConfigImpl.cpp',
     'QueryConfigImpl.cpp', 'AssetTraverser.cpp',
     'geAssetRoot.cpp', 'SearchTabReferenceImpl.cpp',
-    'JsonUtils.cpp', 'JsUtils.cpp'
+    'JsonUtils.cpp', 'JsonUtilsV1.cpp', 'JsUtils.cpp'
```

Comment:
Just for clarity I'll ask this: Doing the V1 version of `JsonUtils.cpp` isn't really needed to fix issue #622, correct?

Reply:
That's correct. I left it in there from my previous attempt, but I will remove it.